src/screens/Components.js

```diff
@@ -284,6 +284,100 @@ export default class Components extends React.Component {
                 >
                   <LinearGradient colors={['transparent', 'rgba(0,0,0, 0.8)']} style={styles.cardGradient} />
                 </Card>
+                <Block flex style={{ width: '100%' }}>
+                  <Block flex style={{ padding: theme.SIZES.BASE }}>
+                    <Text h5>Sliders</Text>
+                  </Block>
+                  <Slider
+                    trackStyle={styles.slider}
+                    step={1}
+                    minimumValue={0}
+                    maximumValue={71}
+                  />
+                  <Slider
+                    trackStyle={styles.slider}
+                    step={1}
+                    thumbStyle={{ borderColor: '#fe2272' }}
+                    activeColor="#fe2272"
+                    minimumValue={0}
+                    maximumValue={71}
+                  />
+                  <Slider
+                    trackStyle={styles.slider}
+                    step={1}
+                    thumbStyle={{ borderColor: '#46df31' }}
+                    activeColor="#46df31"
+                    minimumValue={0}
+                    maximumValue={71}
+                  />
+                  <Block />
+                  <Block flex>
+                    <Block flex style={{ padding: theme.SIZES.BASE }}>
+                      <Text h5>Radio</Text>
+                    </Block>
+
+                    <Radio
+                      containerStyle={styles.radio}
+                      label="Galio"
+                    />
+                    <Radio
+                      containerStyle={styles.radio}
+                      color="#fe2472"
+                      label="is"
+                    />
+                    <Radio
+                      containerStyle={styles.radio}
+                      color="#45df32"
+                      label="awesome!"
+                    />
+                  </Block>
+                  {/* color on checkboxStyle prop is not working */}
+                  <Block flex>
+                    <Block flex style={{ padding: theme.SIZES.BASE }}>
+                      <Text h5>Checkbox</Text>
+                    </Block>
+                    <Checkbox
+                      style={styles.checkbox}
+                      label="Galio"
+                    />
+                    <Checkbox
+                      style={styles.checkbox}
+                      label="is"
+                      checkboxStyle={{
+                        borderColor: '#fe2472',
+                      }}
+                    />
+                    <Checkbox
+                      style={styles.checkbox}
+                      label="awesome!"
+                      checkboxStyle={{
+                        borderColor: '#45df32',
+                      }}
+                    />
+                  </Block>
+                  {/* color & trackColor is not working.. */}
+                  <Block felx>
```

Comment:
The text isn't aligned in the same way the other texts are, you could use as prop for the this Block`style={{ padding: theme.SIZES.BASE }}`